test/metabase/api/embed_test.clj

```diff
@@ -257,58 +284,58 @@
 
 (deftest card-locked-params-test
   (with-embedding-enabled-and-new-secret-key
-    (with-temp-card [card {:enable_embedding true, :embedding_params {:abc "locked"}}]
+    (with-temp-card [card {:enable_embedding true, :embedding_params {:venue_id "locked"}}]
```

Comment:
Since the tests now use *real* parameters I switched them from `:abc` (totally fake) to `:venue_id` which actually makes sense given the query for the mock Card created by `with-temp-card`